k8s/ingress.yml

```
@@ -2,28 +2,31 @@ apiVersion: networking.k8s.io/v1
     nginx.ingress.kubernetes.io/rewrite-target: /
     nginx.ingress.kubernetes.io/ssl-redirect: "false"
 spec:
   rules:
-  - host: chat-tws.com
+  - host: chat-sunil.com
     http:
       paths:
-      - pathType: Prefix
+      - path: /frontend
+        pathType: Prefix
         path: "/"
```

YAMLlint (1.37.1)

[error] duplication of key "path" in mapping

(key-duplicates)